esbuild.js

```diff
@@ -136,21 +136,4 @@ const config = {
     define,
 };
 
-esbuild
-    .build(config)
-    .then(() => {
-        if (!isProd) {
-            return;
-        }
-
-        // minification of the details-view bundle can lead to a collision with the axe-core
-        // package, so we do not minify the identifiers specifically for the details-view bundle.
-        config.entryPoints = {
-            detailsView: `${src}/DetailsView/details-view-initializer.ts`,
-        };
-        config.minify = false;
-        config.minifyWhitespace = true;
-        config.minifySyntax = true;
-        esbuild.build(config).catch(console.error);
-    })
-    .catch(console.error);
+esbuild.build(config).catch(console.error);
```

Comment:
I think this needs to *both* emit to console.error *and* invoke `process.exit(nonzero)` - right now, build failures don't actually cause webpack (or a build pipeline) to bail